sm-core/src/main/java/com/salesmanager/core/business/modules/cms/category/CategoryImagePut.java

```diff
@@ -0,0 +1,16 @@
+package com.salesmanager.core.business.modules.cms.category;
```

Comment:
what's this interface for?

Reply:
It's for saving the image in the storage, it's implemented in each storage type, in our case only infinspan